packages/astro/client.d.ts

```diff
@@ -2,12 +2,6 @@
 /// <reference path="./types/content.d.ts" />
 /// <reference path="./types/actions.d.ts" />
 
-// eslint-disable-next-line  @typescript-eslint/no-namespace
-declare namespace App {
-	// eslint-disable-next-line  @typescript-eslint/no-empty-interface
-	export interface Locals {}
-}
-
```

Comment:
This is unnecessary, the default definition is already that. However, it was done in a `.d.ts` file, which don't get sent to `dist` so it never worked.